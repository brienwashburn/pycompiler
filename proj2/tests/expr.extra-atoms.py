10
10.10
"hello"
b"hello"
True
False
None
...
Ellipsis
a.b.c
(a + 10).b.c
