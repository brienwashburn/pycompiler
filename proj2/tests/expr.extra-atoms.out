(Module
 (Expr (Num 10))
 (Expr (Num 10.1))
 (Expr (Str "hello"))
 (Expr (Bytes #"hello"))
 (Expr (NameConstant True))
 (Expr (NameConstant False))
 (Expr (NameConstant None))
 (Expr (Ellipsis))
 (Expr (Name Ellipsis))
 (Expr (Attribute (Attribute (Name a) b) c))
 (Expr (Attribute (Attribute (BinOp (Name a) Add (Num 10)) b) c)))
